packages/sandbox/src/helpers/postMessageRelayHelpers.ts

```diff
@@ -106,6 +132,10 @@ export type IncomingEmbedMessage = MessageEvent;
 //     name: typeof EXPLORER_LISTENING_FOR_SCHEMA;
 //   }>
 // | MessageEvent<{
+//     name: typeof EXPLORER_CHANGE_PROTOCOL;
```

Comment:
annoying but can we add the other messages here & add this message to the other `postMessageRelayHelper` file please. I think we should be able to upgrade TS in this branch now, so these don't need to be comments 🥳  - I'll open a PR in a bit